### PR TITLE
New README

### PR DESCRIPTION
**Pull request by ethanwharris**
## What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/lightning-flash/tree/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests? [not needed for typos/docs]
- [ ] Did you verify new and existing tests pass locally with your changes?
- [ ] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/lightning-flash/blob/master/CHANGELOG.md)?

## PR review
 - [ ] Is this pull request ready for review? (if not, please submit in draft mode)

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,558 +60,49 @@ Pip / conda
...
 
-## What is Flash
-Flash is a framework of tasks for fast prototyping, baselining, finetuning and solving business and scientific problems with deep learning. It is focused on:
-
-- Predictions
-- Finetuning
-- Task-based training
-
-It is built for data scientists, machine learning practitioners, and applied researchers.
+## What is Flash / Setting up Your Recipe
```
Comment (tchaton) [inline README.md:67]:
```suggestion
## Flash, complex PyTorch Recipes made simple
```

File: flash/image/classification/transforms.py [hunk 2]
```diff
@@ -28,6 +29,22 @@
...
 
+if _ALBUMENTATIONS_AVAILABLE:
+    import albumentations
+
+
+class AlbumentationsAdapter(torch.nn.Module):
+    def __init__(self, transform):
+        super().__init__()
+        if not _ALBUMENTATIONS_AVAILABLE:
+            raise MisconfigurationException("Please, run `pip install albumentations`")
```
Comment (ethanwharris) [inline flash/image/classification/transforms.py:40]: I think we can use `flash.core.utilities.imports.requires` here

File: tests/image/classification/test_data.py [hunk 3]
```diff
@@ -569,3 +574,41 @@ def test_from_bad_csv_no_image(bad_csv_no_image):
             num_workers=0,
         )
         _ = next(iter(img_data.train_dataloader()))
+
+
+@pytest.mark.skipif(
+    not (_IMAGE_TESTING and _ALBUMENTATIONS_AVAILABLE), reason="image libraries + albumentation aren't installed."
```
Comment (ethanwharris) [inline tests/image/classification/test_data.py:580]: Should just check `_IMAGE_AVAILABLE` here as I think albumentations will only be installed for the image_extras testing

File: README.md [hunk 4]
```diff
@@ -25,597 +21,235 @@
...
-## What is Flash
-Flash is a framework of tasks for fast prototyping, baselining, finetuning and solving business and scientific problems with deep learning. It is focused on:
-
-- Predictions
-- Finetuning
-- Task-based training
+See [our installation guide](https://lightning-flash.readthedocs.io/en/latest/installation.html) for more options.
 
-It is built for data scientists, machine learning practitioners, and applied researchers.
+## Flash in 4 steps
```
Comment (ethanwharris) [inline README.md:55]: I think that 3 steps would be better here, then have predictions / serving as additional stuff later on in the readme similar to augmentations

Comment (tchaton) [reply README.md:55]: Done !

File: tests/image/classification/test_data.py (hunk 3 above)
Comment (tchaton) [reply tests/image/classification/test_data.py:580]: albumentations isn't a dependency. This is just for demo purpose.

Comment (ethanwharris) [reply tests/image/classification/test_data.py:580]: Seems weird to me to have a test that we don't expect to run

File: README.md [hunk 5]
```diff
@@ -25,597 +21,233 @@
...
-- Jul 1: [Lightning Flash 0.4](https://devblog.pytorchlightning.ai/lightning-flash-0-4-flash-serve-fiftyone-multi-label-text-classification-and-jit-support-97428276c06f)
-- Jun 22: [Ushering in the New Age of Video Understanding with PyTorch](https://medium.com/pytorch/ushering-in-the-new-age-of-video-understanding-with-pytorch-1d85078e8015)
-- May 24: [Lightning Flash 0.3](https://devblog.pytorchlightning.ai/lightning-flash-0-3-new-tasks-visualization-tools-data-pipeline-and-flash-registry-api-1e236ba9530)
-- May 20: [Video Understanding with PyTorch](https://towardsdatascience.com/video-understanding-made-simple-with-pytorch-video-and-lightning-flash-c7d65583c37e)
-- Feb 2: [Read our launch blogpost](https://pytorch-lightning.medium.com/introducing-lightning-flash-the-fastest-way-to-get-started-with-deep-learning-202f196b3b98)
+<div align="center">
+  <b>
+    Flash enables you to easily configure and run complex AI recipes for over 15 tasks across 7 data domains
+  </b>
+</div>
```
Comment (SeanNaren) [inline README.md:37]: Is it necessary for this to be bold? I think we can get away with this being lower-cased

File: README.md [hunk 6]
```diff
@@ -25,597 +21,233 @@
...
 
----
+<div align="center">
+  <a href="https://lightning-flash.readthedocs.io/en/stable">
+    <img src="https://pl-flash-data.s3.amazonaws.com/assets/banner.gif">
+  </a>
+</div>
 
-## Installation
+## Getting stated
```
Comment (SeanNaren) [inline README.md:45]:
```suggestion
## Getting started
```

File: README.md [hunk 7]
```diff
@@ -25,597 +21,233 @@
...
+<div align="center">
+<img src="/docs/source/_static/images/flash_zero.gif?raw=true" width="75%">
+</div>
 
-# data
-dataset = datasets.MNIST("./data_folder", download=True, transform=transforms.ToTensor())
-train, val = random_split(dataset, [55000, 5000])
+Flash Zero is a zero-code machine learning platform built
+directly into lightning-flash
+and over [`lightning CLI`](https://pytorch-lightning.readthedocs.io/en/stable/common/lightning_cli.html).
```
Comment (SeanNaren) [inline README.md:202]:
```suggestion
using the [`Lightning CLI`](https://pytorch-lightning.readthedocs.io/en/stable/common/lightning_cli.html).
```

File: README.md [hunk 8]
```diff
@@ -25,597 +21,233 @@
...
+To get started and view the available tasks, run:
 
-# train
-flash.Trainer().fit(classifier, DataLoader(train), DataLoader(val))
+```py
+  flash --help
 ```
 
-## Infinitely customizable
+For example, to train an image classifier for 10 epochs with a `resnet50` backbone on 2 gpus over you own data, you can do:
```
Comment (SeanNaren) [inline README.md:210]:
```suggestion
For example to train an image classifier for 10 epochs with a `resnet50` backbone on 2 GPUs using you own data, you can do:
```

File: README.md [hunk 9]
```diff
@@ -25,597 +21,233 @@
...
 
 ---
 
-## Tasks
-Flash is built as a collection of community-built tasks. A task is highly opinionated and laser-focused on solving a single problem well, using state-of-the-art methods.
+## PyTorch Recipes
 
-### Example 1: Image embedding
-Flash has an [Image Embedder task](https://lightning-flash.readthedocs.io/en/latest/reference/image_embedder.html) to encode an image into a vector of image features which can be used for anything like clustering, similarity search or classification.
+### Make predictions with Flash !
```
Comment (ethanwharris) [inline README.md:112]:
```suggestion
### Make predictions with Flash!
```

File: README.md (hunk 8 above)
Comment (tchaton) [reply README.md:210]: Done !

File: README.md (hunk 5 above)
Comment (tchaton) [reply README.md]: Removed !

**Pull request merged**